class-apporprepp-admin.php

```diff
@@ -65,6 +65,25 @@ public function register_fields() {
 				array( 'post_type' => $post_type )
 			);
 
+			if ( $post_type === "post" ) {
```

Comment:
Please, use single quotes 
```suggestion
			if ( $post_type === 'post' ) {
```

In any case, why did you restrict the option for posts? It could be handy for other post types.